docker-compose.test.yml

```diff
@@ -23,128 +23,122 @@
 version: '2'
 
 services:
-  service_base:
+  test-service_base:
     # Overrides default inspirehep config.
     extends:
       file: services.yml
       service: base
     environment:
-      - APP_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://inspirehep:dbpass123@database:5432/inspirehep
-      - APP_BROKER_URL=amqp://guest:guest@rabbitmq:5672//
-      - APP_CELERY_RESULT_BACKEND=amqp://guest:guest@rabbitmq:5672//
-      - APP_CACHE_REDIS_URL=redis://redis:6379/0
-      - APP_ACCOUNTS_SESSION_REDIS_URL=redis://redis:6379/2
-      - APP_SEARCH_ELASTIC_HOSTS=indexer
+      - APP_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://inspirehep:dbpass123@test-database:5432/inspirehep
+      - APP_BROKER_URL=amqp://guest:guest@test-rabbitmq:5672//
+      - APP_CELERY_RESULT_BACKEND=amqp://guest:guest@test-rabbitmq:5672//
+      - APP_CACHE_REDIS_URL=redis://test-redis:6379/0
+      - APP_ACCOUNTS_SESSION_REDIS_URL=redis://test-redis:6379/2
+      - APP_SEARCH_ELASTIC_HOSTS=test-indexer
   unit:
     extends:
-      service: service_base
+      service: test-service_base
     volumes_from:
-      - static
+      - test-static
     command: py.test inspirehep tests/unit
   disambiguation:
     extends:
-      service: service_base
+      service: test-service_base
     command: py.test tests/integration/disambiguation
     volumes_from:
-      - static
+      - test-static
     links:
-      - database
-      - indexer
-      - rabbitmq
-      - redis
+      - test-database
+      - test-indexer
+      - test-rabbitmq
+      - test-redis
     depends_on:
-      - worker
+      - test-worker
   workflows:
     extends:
-      service: service_base
+      service: test-service_base
     command: py.test tests/integration/workflows
     volumes_from:
-      - static
+      - test-static
     links:
-      - database
-      - indexer
-      - rabbitmq
-      - redis
+      - test-database
+      - test-indexer
+      - test-rabbitmq
+      - test-redis
     depends_on:
-      - worker
-  integration:
+      - test-worker
+  test-integration:
```

Comment:
This looks like it's causing https://travis-ci.org/inspirehep/inspire-next/jobs/212426038.